core/sql/sqlcomp/CmpSeabaseDDLroutine.cpp

```diff
@@ -839,21 +1399,27 @@ void CmpSeabaseDDL::createSeabaseRoutine(
 
   char * ptr = NULL;
   Lng32 len = 0;
+  
   cliInterface.getPtrAndLen(1, ptr, len);
   str_cpy_all(externalPath, ptr, len);
   externalPath[len] = '\0'; 
-
+  
+  cliInterface.getPtrAndLen(2, ptr, len);
+  str_cpy_all(libBlobHandle, ptr, len);
+  libBlobHandle[len] = '\0'; 
+    
+  
+  NAString extPath(externalPath);
+  size_t lastDot = extPath.last('.');
+     
+  
+  if (lastDot != NA_NPOS)  
```

Comment:
Here is another place that would potentially fail if the library name contained an embedded period.  Should an internal error be generated if there is no period in the name?